modules/integration/tests-integration/tests-backend/src/test/resources/artifacts/AM/synapseconfigs/rest/dummy_api_APIM5474.xml

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definitions xmlns="http://ws.apache.org/ns/synapse">
+    <sequence name="fault">
+        <log level="custom">
+            <property name="STATUS" value="Executing default 'fault' sequence"/>
+            <property name="ERROR_CODE" expression="get-property('ERROR_CODE')"/>
+            <property name="ERROR_MESSAGE" expression="get-property('ERROR_MESSAGE')"/>
+        </log>
+        <filter source="get-property('MESSAGE_FORMAT')" regex="soap1[1-2]">
+            <then>
+                <property name="SOAP_FAULT_CODE" value="Server"/>
+                <makefault>
+                    <code expression="$ctx:SOAP_FAULT_CODE"/>
+                    <reason expression="$ctx:ERROR_MESSAGE"/>
+                </makefault>
+            </then>
+            <else>
+                <payloadFactory>
+                    <format>
+                        <am:fault xmlns:am="http://wso2.org/apimanager">
+                            <am:code>$1</am:code>
+                            <am:type>Status report</am:type>
+                            <am:message>Runtime Error</am:message>
+                            <am:description>$2</am:description>
+                        </am:fault>
+                    </format>
+                    <args>
+                        <arg expression="$ctx:ERROR_CODE"/>
+                        <arg expression="$ctx:ERROR_MESSAGE"/>
+                    </args>
+                </payloadFactory>
+                <filter source="$axis2:HTTP_METHOD" regex="^(?!.*(POST|PUT)).*$">
+                    <property name="messageType" value="application/xml" scope="axis2"/>
+                </filter>
+            </else>
+        </filter>
+        <filter xpath="$ctx:CUSTOM_HTTP_SC">
+            <then>
+                <property name="HTTP_SC" expression="$ctx:CUSTOM_HTTP_SC" scope="axis2"/>
+            </then>
+            <else>
+                <property name="HTTP_SC" value="500" scope="axis2"/>
+            </else>
+        </filter>
+        <class name="org.wso2.carbon.apimgt.usage.publisher.APIMgtFaultHandler"/>
```

Comment:
We moved handlers to gateway component. So this should be `org.wso2.carbon.apimgt.gateway.handlers.analytics.APIMgtFaultHandler`
Please update.

Reply:
@manoramahp Please fix this.

Reply:
Done